dojo_toolkit/dojo.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import time
+import threading
+
+from watchdog.observers import Observer
+
+from dojo_toolkit.code_handler import DojoCodeHandler
+from dojo_toolkit.notifier import GnomeNotifier
+from dojo_toolkit.test_runner import DoctestTestRunner
+from dojo_toolkit.timer import dojo_timer
+
+
+class Dojo:
```

Comment:
I believe building API docs should go to another PR, specially because this is only the first round of refactor that will happen to this project. 

Unless there is code that is too hard to understand (which should be rewritten).